### PR TITLE
Adding picatrix docker container to the development docker config

### PR DESCRIPTION
**Pull request by kiddinn**
This adds a jupyter notebook docker container, based on picatrix to the dev docker config.

This means that once you use the dev docker config a jupyter notebook with access to the Timesketch instance

### CODE REVIEW

**Comment by kiddinn** [comment]
Wait with a review, need to add one more change in before

**Comment by kiddinn** [comment]
@berggren I've added the first version of the snippets support, please review

File: docker/dev/build/timesketch_token [hunk 1]
```diff
@@ -0,0 +1,2 @@
+��^_~&l�/�
```
Comment (berggren) [inline docker/dev/build/timesketch_token:1]: Is this token suppose to be checked in?

**Review by berggren — COMMENTED** [review]
One question, ptal

File: docker/dev/build/timesketch_token (hunk 1 above)
Comment (kiddinn) [reply docker/dev/build/timesketch_token:1]: Yes, this is the dev password

**Review by kiddinn — COMMENTED** [review]
Back to you

**Review by berggren — APPROVED** [review]
LGTM

**Pull request merged**